oslabs-beta/Kubiquity
# Move state to App component

WHAT DOES THIS PR DO? 
1. Adds in a new frontend component
2. Connects new frontend component to backend

HOW TO TEST THIS PR:
1. Run `gh pr checkout 22 && npm install && npm run dev`
2. After Webpack is rebuilt, the server is running, and the app is connected to Mongo, refresh the application and verify that the testing version of the app works and looks good. 
3. If everything looks good, open kill the Electron app and open VS Code. Follow the instructions in the App component for using a real K8s cluster instead of our test data. 
4. Open Docker
5. Run `minikube start`
6. Run `helm upgrade -i prometheus prometheus-community/prometheus --namespace prometheus`
7. Expose the port with `export POD_NAME=$(kubectl get pods --namespace prometheus -l "app=prometheus,component=server" -o jsonpath="{.items[0].metadata.name}") && kubectl --namespace prometheus port-forward $POD_NAME 9090`
8. Run `npm run dev`
9. After Webpack is rebuilt, the server is running, and the app is connected to Mongo, refresh the application and verify that the testing version of the app works and looks good. 

ADDITIONAL NOTES: 
1. Hopefully last big PR before MVP presentations, so definitely make sure this works for you. 
2. Make sure the timing for the Splash page still works. 
3. 🙏
